PyPDF2/_page.py

```diff
@@ -356,6 +365,51 @@ def createBlankPage(
         deprecate_with_replacement("createBlankPage", "create_blank_page")
         return PageObject.create_blank_page(pdf, width, height)
 
+    @property
+    def rotation(self) -> int:
+        """
+        A read/write number giving the page VISUAL rotation will be within (0,90,180,270)
+        can be used to rotate the page with += or -= operators
+        this does not affect "/Contents"
+        """
+        return int(self.get(PG.ROTATE, 0))
+
+    @rotation.setter
+    def rotation(self, r: Union[int, float]) -> None:
+        self[NameObject(PG.ROTATE)] = NumberObject((((int(r) + 45) // 90) * 90) % 360)
```

Comment:
Why is this transformation necessary?

I thought it would be as simple as:

```suggestion
        self[NameObject(PG.ROTATE)] = NumberObject(int(r) % 360)
```

Reply:
Ah, got it: You automatically round to the closest multiple of 90!

Reply:
my concern was about possible float with invalid rounding (eg after calculration you get 90.000001)
your opinion ?

Reply:
Right, yes, that could happen. I think it's a good solution - awesome work :+1: